src/components/countdownTimer/CountdownTimer.tsx

```diff
@@ -15,7 +15,7 @@ export const CountdownTimer = ({
   intervalSeconds = 1,
 }: CountdownTimerProps) => {
   const now = dayjs().unix();
-  const [countdown, setCountdown] = useState(date - now);
+  const [countdown, setCountdown] = useState(date ? date - now : -1);
```

Comment:
I believe it's more common to use milliseconds and it can lead to unnecessary confusion. Why does `now` have to be in seconds? Can't `formatTime` be made to handle milliseconds?